### ISSUE TITLE
Response is locked

### ISSUE THREAD

**Issue by Buszmen** [zellwk/zl-fetch#2]
Could you return cloned version of the `Response` object before reading it? I'd really like to use it but I get an error:
```
TypeError: Failed to execute 'clone' on 'Response': Response body is already used
    at zl_fetch__WEBPACK_IMPORTED_MODULE_0___default.then.result
```
which means that it has already been read and it's locked. It must be done as early as possible so inside the library using this: https://developer.mozilla.org/en-US/docs/Web/API/Response/clone

**Comment by zellwk** [comment]
Are you using this on the frontend or in Node? 

It looks like support isn't solid.

**Comment by Buszmen** [comment]
On frontend

**Comment by zellwk** [comment]
@Buszmen This is fixed in [v2.2.0](https://github.com/zellwk/zl-fetch/tree/v2.2.0)
